app/src/main/java/org/oppia/app/home/HomeFragmentController.kt

```diff
@@ -1,11 +1,15 @@
 package org.oppia.app.home
 
+import android.content.Intent
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
+import androidx.core.content.ContextCompat.startActivity
 import androidx.fragment.app.Fragment
+import kotlinx.android.synthetic.main.home_fragment.view.*
```

Comment:
Is this import needed? Also, prefer not to use '*' imports since it pollutes the global symbol namespace (though I know Android Studio does this automatically despite being configured not to).

Reply:
I will remove them.